.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,67 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ main ]
+  schedule:
+    - cron: '33 22 * * 1'
```

Comment:
seems a bit odd scheduled time? https://crontab.guru/#33_22_*_*_1

Reply:
This is an example setup I'm using from https://github.com/City-of-Helsinki/kuva-demo-project. I think it is just to run it once a week, I don't know if the time matters.

Reply:
This could be more convenient: https://crontab.guru/every-week